src/idl_gen_python.cpp

```diff
@@ -589,18 +592,32 @@ class PythonStubGenerator {
       }
     }
 
+    // Remove imports from exports
+    for (const Import &import : imports.exports) {
+      if (import.name == "") {
+        modules.erase(import.module);
+      } else {
+        if (auto search = names_by_module.find(import.module);
```

Comment:
`  D:\a\flatbuffers\flatbuffers\src\idl_gen_python.cpp(600,1): error C2429: language feature 'init-statements in if/switch' requires compiler flag '/std:c++17'`

Reply:
i thought we had access to cpp17 but i will look for a solution which also works without :+1:

Reply:
done.